NotificationService/NotificationService.BusinessLibrary/Business/v1/EmailManager.cs

```diff
@@ -88,6 +96,7 @@ public class EmailManager : IEmailManager
             this.encryptionService = encryptionService;
             this.templateManager = templateManager;
             this.templateMerge = templateMerge;
+            this.cloudStorageClient = cloudStorageClient;
```

Comment:
why is this required here

Reply:
This can be removed. Had added it from a previous change. I'll remove it in the next iteration.